org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/target/editor/internal/MavenAccessor.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Patrick Ziegler and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *      Patrick Ziegler - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.m2e.pde.ui.target.editor.internal;
+
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+import org.eclipse.m2e.pde.target.MavenTargetDependency;
+
+/**
+ * <p>
+ * This class provides a simply way to access/modify the fields of a Maven
+ * dependency. Available fields are:
+ * <ul>
+ * <li>group id</li>
+ * <li>artifact id</li>
+ * <li>version</li>
+ * <li>classifier</li>
+ * <li>type</li>
+ * </ul>
+ * </p>
+ */
+public class MavenAccessor implements Function<Object, String>, BiConsumer<Object, Object> {
```

Comment:
Object seems not really they type to use here but `MavenTargetDependency` / `String`?